Shopify/kubeaudit
# Mount docker sock check feature

##### Description

Fixes #216

Changes:
- Added a new check to the CLI, mountds
- 'kubeaudit all' also runs mountds as well
- Added a new label 'allow-mount-docker-sock' to override the check
- New error objects to handle both the check and the override
- Changed the config to deny this override by default


##### Type of change

[✓] New feature :sparkles:
[✓] This change requires a documentation update :book:

##### How Has This Been Tested?

[✓] Regression tests passing
[✓] New unit tests:
  - Check that container mounting /var/run/docker.sock shows a warning
  - Check that the label overrides the check with a warning (pod + container level)

  
##### Checklist:

[✓] I have :tophat: my changes (A 🎩 specifically includes pulling down changes, setting them up, and manually testing the changed features and potential side effects to make sure nothing is broken)
[✓] I have performed a self-review of my own code
[✓] I have made corresponding changes to the documentation
[✓] I have added tests that prove my fix is effective or that my feature works
[✓] New and existing unit tests pass locally with my changes
[✓] The test coverage did not decrease
